slam_vdb_ros/include/slam_vdb_ros/slam_vdb_ros.hpp

```diff
@@ -2,18 +2,15 @@
 #define SLAM_VDB__SLAM_VDB_HPP_
```

Comment:
lets update the header guards accordingly

Reply:
Updated